tabs/ansible/jenkins/security_group/aws.yaml

```diff
@@ -10,20 +10,23 @@ imports:
 inputs:
 
   rg_id:
-    description: Mocked Resource group ID
-    display_label: mocked resource group id
+    description: Resource group ID
+    display_label: Resource Group ID
     type: string
```

Comment:
Please validate for the rest blueprints if the inputs are in use

Reply:
I resolved it by passing deployment id to security group blueprints and fetch needed parameters from deployment id of parent.